README.md

```diff
@@ -78,6 +78,8 @@ Optional parameters
 * `tag`: (string) - unique identifier to stop duplicate notifications
 * `lang`: (string) - BCP 47 language tag for the notification (default: `en`)
 * `timeout`: (integer) - number of seconds to close the notification automatically
+* `requireInteraction`: (boolean) - keep the notification open indefinitely
+* `closeOnClick`: (boolean) - close the notification when clicked
```

Comment:
Let's add a note here that this can be useful for Chrome, which does not close notifications on click by default (Only the little (x) closes the notification on Chome).